ui/src/components/Player/DetectionDialog.tsx

```diff
@@ -126,9 +129,44 @@ export default function DetectionDialog({
         onClose={handleClose}
         aria-labelledby="form-dialog-title"
       >
-        <DialogTitle id="form-dialog-title">
-          {!submitted ? "Report what you heard" : "Thanks for submitting!"}
-        </DialogTitle>
+        <IconButton
+          aria-label="close"
+          onClick={handleClose}
+          sx={{
+            position: "absolute",
+            right: 8,
+            top: 8,
+            color: (theme) => theme.palette.grey[500],
```

Comment:
@paulcretu Thanks for letting me know! What do you mean it will auto-complete because it has typescript info?

Reply:
If you use VS Code or an editor with typescript support, you should get auto-completion / intellisense 
<img width="791" alt="image" src="https://github.com/orcasound/orcasite/assets/1069628/9e5fbb6a-b661-440a-97b6-886c5f3b12da">


Which won't happen with the shorthand since it's just a string

Reply:
@paulcretu ah I see, thanks!